src/main/java/org/colonelkai/guielements/nodes/downloadslist/DownloadBox.java

```diff
@@ -86,14 +90,36 @@ private VBox topVBox() {
         return box;
     }
 
+
+    private Button getCancelButton() {
+        Button cancelButton = new Button("X");
+        cancelButton.setFont(this.fontSmall);
+        cancelButton.setAlignment(Pos.CENTER);
+        cancelButton.prefHeightProperty().bind(this.vbox.heightProperty());
+
+        cancelButton.setOnAction(actionEvent -> {
+            this.downloadContext.getAsynced().interrupt();
```

Comment:
This starts another download and then cancels that download. This doesn't cancel the original download.

Reply:
Oh.